core/src/main/java/org/elasticsearch/action/get/MultiGetResponse.java

```diff
@@ -173,4 +173,17 @@ public void writeTo(StreamOutput out) throws IOException {
             response.writeTo(out);
         }
     }
+    
+    @Override
+    public String toString() {
+        try {
+            XContentBuilder builder = XContentFactory.jsonBuilder().prettyPrint();
+            builder.startObject();
+            toXContent(builder, EMPTY_PARAMS);
+            builder.endObject();
+            return builder.string();
+        } catch (IOException e) {
+            return "{ \"error\" : \"" + e.getMessage() + "\"}";
+        }
+    }
```

Comment:
You can simply use `Strings.toString(this)` instead, at least that's what e.g. GetResponse does.